corehq/couchapps/last_modified/views/by_last_modified/map.js

```diff
@@ -1,12 +1,9 @@
 function(doc){
-    if (doc.doc_type == "Group" || doc.doc_type == "Group-Deleted") {
+    if (doc.doc_type === "Group" || doc.doc_type === "Group-Deleted") {
         emit([doc.doc_type, doc.last_modified], null);
-    } else if (doc.doc_type == "Domain" || doc.doc_type == "Domain-Deleted") {
-        emit([doc.doc_type, doc.last_modified], null);
-    } else if (doc.doc_type == "WebUser" ||
-               doc.doc_type == "WebUser-Deleted" ||
-               doc.doc_type == "CommCareUser" ||
-               doc.doc_type == "CommCareUser-Deleted") {
+    } else if (doc.doc_type === "Domain" || doc.doc_type === "Domain-Deleted") {
         emit([doc.doc_type, doc.last_modified], null);
+    } else if (doc.base_doc && (doc.base_doc === "CouchUser" || doc.doc_type === "CouchUser-Deleted")) {
```

Comment:
:man_facepalming: clearly can't write code when there are no tests. making some for when i create the user staging table